io/buffer.ts

```diff
@@ -352,7 +352,9 @@ export class Buffer implements Writer, WriterSync, Reader, ReaderSync {
       // don't spend all our time copying.
       copy(this.#buf.subarray(this.#off), this.#buf);
     } else if (c + n > MAX_SIZE) {
-      throw new Error("The buffer cannot be grown beyond the maximum size.");
+      throw new Error(
+        'The buffer cannot be grown beyond the maximum size of "${MAX_SIZE}"',
```

Comment:
```suggestion
        `The buffer cannot be grown beyond the maximum size of "${MAX_SIZE}"`,
```

Reply:
Yikes, sorry about that. I'll update the PR.